core/dungeon-masters-guide/races/race-aasimar.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<elements>
+	<info>
+		<name>Aasimar</name>
+		<description>Aasimar example race from the Dungeon Master’s Guide</description>
+		<author url="http://dnd.wizards.com">Wizards of the Coast</author>
+		<update version="0.0.1">
+			<file name="race-aasimar.xml" url="https://raw.githubusercontent.com/AuroraLegacy/elements/master/dungeon-masters-guide/races/race-aasimar.xml" />
+		</update>
+	</info>
+
+	<element name="Aasimar" type="Race" source="Dungeon Master’s Guide" id="ID_WOTC_DMG_RACE_AASIMAR">
+		<description>
+			<p class="indent"><strong><em>Ability Score Increase.</em></strong> Your Charisma score increases by 2 and Wisdom score increases by 1.</p>
```

Comment:
It maybe would make sense to add first paragraph from the Aasimar description from the book here.